### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by varsha-bharath**
Hi I ve added the associations please check it out and merge

### CODE REVIEW

**Comment by Pamphilemkp** [comment]
Hi @varsha-bharath, I see you that you have made some great progress , you are almost there, please make sure to fix the rubocop errors by running this command in your IDE "rubocop --autocorrect" that will fix and then please  push your code to GitHub so can be able to review your code, happy coding

**Comment by varsha-bharath** [comment]
I did that but still it has some conflicts if i redo "rubocop --autocorrect" im getting "75 files inspected, 12 offenses detected" but when i try to commit its saying nothing to commit

File: app/models/parking_slot.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class ParkingSlot < ApplicationRecord
+end
```
Comment (Pamphilemkp) [inline app/models/parking_slot.rb]: Please kindly establish the association relationship between parking slot and the ride according to the diagram

File: app/models/vehicle.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Vehicle < ApplicationRecord
+  has_many :bookings, dependent: :destroy
+  has_many :drivers, dependent: :destroy
+  has_many :rides, dependent: :destroy
+  belongs_to :category
+end
```
Comment (Pamphilemkp) [inline app/models/vehicle.rb:6]: Please kindly also add the relationship with the parking slot as shown in the diagram

**Review by Pamphilemkp — CHANGES_REQUESTED** [review]
## Hi @varsha-bharath , 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next point on the  project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

!image

# Hints
- Please please add the one to many associations between the ride and the parking slot as shown in the diagram above.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by varsha-bharath** [comment]
Associations have been added for parking slot and rides now u can merge i guess

**Review by Pamphilemkp — APPROVED** [review]
Hi @varsha-bharath 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✔️ Descriptive PR 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**@pamphilemkp**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**